lib/home/library/video_library.dart

```diff
@@ -11,7 +11,7 @@ import 'package:intl/intl.dart';
 import 'package:dio/dio.dart';
 import 'package:universal_html/html.dart' as html;
 
-class FavoriteIconButton extends StatefulWidget {
+class FavoriteIconButton extends StatelessWidget {
```

Comment:
Since there aren't any instance variables or `setState` calls, this can be a stateless widget!